dashboards/mysql/mysql-prometheus.json

```diff
@@ -159,184 +169,292 @@
         "widget": {
-          "title": "Full Range Join",
+          "title": "Fsync Operations",
           "xyChart": {
             "chartOptions": {
               "mode": "COLOR"
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "rate(mysql_global_status_select_full_range_join{${Cluster},${Location},${Namespace}}[5m])"
+                  "prometheusQuery": "rate(mysql_global_status_innodb_data_fsyncs[5m])",
+                  "unitOverride": ""
                 }
               }
             ],
-            "timeshiftDuration": "0s",
+            "thresholds": [],
             "yAxis": {
+              "label": "",
               "scale": "LINEAR"
             }
           }
         },
-        "width": 3,
-        "xPos": 3,
-        "yPos": 12
+        "width": 24,
+        "xPos": 24,
+        "yPos": 41
       },
       {
-        "height": 3,
+        "height": 10,
         "widget": {
-          "title": "Range Check",
+          "title": "Insert | Update | Delete Statements",
           "xyChart": {
             "chartOptions": {
               "mode": "COLOR"
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "rate(mysql_global_status_select_range_check{${Cluster},${Location},${Namespace}}[5m])"
+                  "prometheusQuery": "rate(mysql_global_status_commands_total{command=~\"insert|update|delete\"}[5m])",
+                  "unitOverride": ""
                 }
               }
             ],
-            "timeshiftDuration": "0s",
+            "thresholds": [],
             "yAxis": {
+              "label": "",
               "scale": "LINEAR"
             }
           }
         },
-        "width": 3,
-        "xPos": 6,
+        "width": 24,
         "yPos": 12
       },
       {
-        "height": 3,
+        "height": 10,
         "widget": {
-          "title": "Scan",
+          "title": "Select Statements",
           "xyChart": {
             "chartOptions": {
               "mode": "COLOR"
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "rate(mysql_global_status_select_scan{${Cluster},${Location},${Namespace}}[5m])"
+                  "prometheusQuery": "rate(mysql_global_status_commands_total{command=~\"select\"}[5m])",
```

Comment:
Can you do a pass to ensure all the charts are consistently using the template variables? Seems like:
- Select statements
- Install | Update | Delete Statements
- Questions
- Fsync Operations
- InnoDB Buffer ...
aren't, so when I filter down to a single cluster or namespace, these still show the metrics for the entire project.